deduplica_comissao.py

```diff
@@ -0,0 +1,193 @@
+
+    for d in to_delete:
+        d.delete()
+
+    # REMOVE AUTORES DUPLICADOS
+    Autor.objects.filter(id__in=autor_from).delete()
+
+
+def estatisticas():
+    # TODO: verificar se content type é o mesmo para todos os BD
+    content_type = ContentType.objects.get(id=37)
+    tipo = TipoAutor.objects.get(id=2)
+
+    for c in Comissao.objects.all().order_by('nome'):
+        print('-------------------------------------------------------------------')
+        print(c)
+
+        for m in models_comissoes + [UnidadeTramitacao]:
+            model_name = m.__name__
+            total = m.objects.filter(comissao=c).count()
+            print(model_name, total)
+
+        try:
+            autor = Autor.objects.get(content_type=content_type, tipo=tipo, object_id=c.id)
+            for a in models_autor:
+                model_name = a.__name__
+                total = a.objects.filter(autor=autor).count()
+                print(model_name, total)
+        except Exception as e:
+            print(e)
+            pass
+        print('-------------------------------------------------------------------')
+
+
+def delete_comissao(id):
+    # TODO: verificar se content type é o mesmo para todos os BD
+    content_type = ContentType.objects.get(id=37)
+    tipo = TipoAutor.objects.get(id=2)
+    comissao = Comissao.objects.get(id=id)
+
+    for m in models_comissoes:
+        m.objects.filter(comissao=comissao).delete()
+
+    try:
+        autor = Autor.objects.get(content_type=content_type, tipo=tipo, object_id=comissao.id)
+        for a in models_autor:
+            a.objects.filter(autor=autor).delete()
+        autor.delete()
+    except Exception as e:
+        print(e)
+
+    try:
+        unidate_tramitacao = UnidadeTramitacao.objects.get(comissao=comissao)
+        models_tramitacao = [Tramitacao, TramitacaoAdministrativo]
+        for m in models_tramitacao:
+            m.objects.filter(unidade_tramitacao_local=unidate_tramitacao).delete()
+            m.objects.filter(unidade_tramitacao_destino=unidate_tramitacao).delete()
+        unidate_tramitacao.delete()
+    except Exception as e:
+        print(e)
+
+    comissao.delete()
+
+
+def deduplicate_comissao():
+    # TODO: verificar se content type é o mesmo para todos os BD
+    content_type = ContentType.objects.get(id=37)
+    tipo = TipoAutor.objects.get(id=2)
+    comissoes_repetidas = [(c['nome'], c['total']) for c in
+                           Comissao.objects.values('nome').annotate(total=Count('nome')).filter(total__gt=1)]
+    print(comissoes_repetidas)
+    for cr in comissoes_repetidas:
+        comissoes = list(Comissao.objects.filter(nome=cr[0]).order_by('id').values_list('id', flat=True))
+
+        print(cr)
+        print(comissoes)
+
+        ## TODO: PODEM EXISTIR CASOS DA COMISSAO DE DESTINO NAO TER AUTOR, TERIA QUE CRIAR
+        ## TODO: POR ENQUANTO OPTEI POR PEGAR A ULTIMA COMISSAO COM AUTOR CRIADO
+        while True:
+            c = comissoes.pop()
```

Comment:
```suggestion
            c = comissoes.pop(0)
```
Para não criar um loop infinito quando o último elemento da lista comissões não possui autor.